.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: required
 
 language: python
 python:
-  - "2.7"
+  - "3.5"
```

Comment:
Is there a reason we’re not running tests on 3.6, 3.7, and 3.8 too? (The reason tests were only run on 2.7 was that there was no "major" version above or below that was supported, but whipper as of the merging of this PR supports Python 3.6, 3.7, and 3.8 in addition to 3.5.)

Reply:
No, there isn't. The fact is that I'm not that familiar with Travis CI and need to learn how to do this: feel free to open a pull request for this :+1:

Reply:
I’ll make a PR once this PR has been merged. :)